### ISSUE TITLE
Start the match prediction page on the current match date

### ISSUE THREAD

**Issue by rogerlejeune** [xabikos/BrazilWC2014#46]
I tried to find if this was possible, but it seems not to be not possible. Or maybe the wizard knows?

**Comment by xabikos** [comment]
It would be nice to have but from a brief investigation I have done it's not easy. Maybe it's a better idea to make clickable the upcoming matches from the home page

**Comment by rogerlejeune** [comment]
I totally agree. We could also actionlink the matches in the upcoming matches to the right pages indeed. That's much easier too!

**Comment by xabikos** [comment]
Go ahead then for the solution you proposed and let me know if you face any problem. You could easily add more than two matches as well

**Comment by rogerlejeune** [comment]
I will! I'll do it tomorrow, then you can check my MVC skillz. :)
